workflow-config/config-jvm/gradle.properties

```diff
@@ -0,0 +1,3 @@
+POM_ARTIFACT_ID=workflow-config-jvm
+POM_NAME=Workflow Runtime JVM Configuration
+POM_PACKAGING=aar
```

Comment:
This should be a `jar` and the `artifacts.json` will need to be updated.

And I'm realizing that the artifacts plugin should probably have a check which compares the packaging to the module type.

Reply:
Whoops. I think when I changed them I changed the jvm one to a jar rather than vice versa.